docs/netlify.toml

```diff
@@ -416,12 +416,12 @@
 
 [[redirects]]
   from = "/manuals/administrator/ci"
-  to = "/integrations"
+  to = "/ci/integrations"
```

Comment:
The global /integrations -> /ci/integrations redirect covers this today, but makes sense to collapse that chain.